.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[profile.release]
+strip = "symbols"
```

Comment:
Why do we need that?

Reply:
I followed the approach used by runwasi. During the release build, Cargo strips the symbol information from the final executable. This practice usually leads to a smaller binary size, which is advantageous for reducing memory footprint and potentially increasing security since it removes details about the program’s internal structure.

Reply:
@cuisongliu Good idea, but please separate an PR from this PR 🙏

Reply:
https://github.com/containers/youki/pull/2503